edgelet/hsm-sys/azure-iot-hsm-c/tests/edge_openssl_pki_ut/edge_openssl_pki_ut.c

```diff
@@ -2092,6 +2121,8 @@ BEGIN_TEST_SUITE(edge_openssl_pki_unittests)
         REGISTER_GLOBAL_MOCK_FAIL_RETURN(X509_add_ext, 0);
 
         REGISTER_GLOBAL_MOCK_HOOK(X509_EXTENSION_free, test_hook_X509_EXTENSION_free);
+
+        REGISTER_GLOBAL_MOCK_HOOK(get_san_entries, test_hook_get_san_entries);
```

Comment:
>REGISTER_GLOBAL_MOCK_HOOK [](start = 8, length = 25)

For consistencies sake, should you call REGISTER_GLOBAL_MOCK_FAIL_RETURN(get_san_entries,NULL) ?

Reply:
Ah so NULL is not a failure because a certificate could not have any SAN entries as these are optional.